qabelbox/src/main/java/de/qabel/qabelbox/box/dto/BoxPath.kt

```diff
@@ -42,5 +41,8 @@ sealed class BoxPath() {
 
 
     fun toList(): List<String> = if (this is Root) emptyList() else parent.toList() + name
+
+    fun toReadable(): String = "/" + toList().joinToString("/")
```

Comment:
This is diverging from https://github.com/Qabel/qabel-core/blob/90145780680b7954f53d068126ec883a5d6e20ff/box/src/main/java/de/qabel/box/storage/dto/BoxPath.kt
Don't know why we have two implementations (probably my bad). Can you switch android to use the core implementation? the BoxPath.toString() from the core implementation is meant to do this (`toReadable`)